docs/steps-rest.md

```diff
@@ -0,0 +1,81 @@
+---
+id: steps-rest
+title: Rest api
+---
+
+# Steps used for testing REST api:
+
+In order to configure url for api please change `apiUrl` field in `functional-tests/kakunin.conf.js`. This will set baseUrl of application api.
```

Comment:
I think you miss a comma here, please add `,` after `API`.

Also, I would change the sentence `This will set baseUrl of application api.` - I would like to avoid `baseUrl` keyword